LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2014 Katrina Owen, _@kytrinyx.com
+Copyright (c) 2017 Exercism, Inc
```

Comment:
It was the only thing I saw... IANAL but that was the only thing I had noticed (a recent attorney's note had mentioned the non-legal distinction of the `(c)` but I can not find that note at the moment, and it is likely based on some outdated things, apparently.)